src/extensions/exception-autocapture/index.ts

```diff
@@ -21,20 +21,19 @@ export class ExceptionObserver {
         this.startIfEnabled()
     }
 
-    get isEnabled() {
+    public get isEnabled(): boolean {
+        if (isBoolean(this.instance.config.capture_exceptions)) {
+            return this.instance.config.capture_exceptions
+        }
         return this.remoteEnabled ?? false
     }
 
-    get isCapturing() {
-        return !!(window?.onerror as any)?.__POSTHOG_INSTRUMENTED__
-    }
-
     get hasHandlers() {
-        return this.originalOnUnhandledRejectionHandler || this.unwrapOnError
+        return !isUndefined(this.unwrapOnError)
```

Comment:
same, is the assumption here if we have one then we have both?

Reply:
Yeah. I guess we could look up either but then would would have to handle the two separately e.g. consider adding a handler to the two functions separately, which would complicate the logic elsewhere. I'm happy to go with this as an assumption for now but certainly willing to reconsider if it causes issues

Reply:
Basing it somewhat off the fact that `isCapturing` previously only tested `window.onerror` anyways